content/blog/2016/2016-08-25-sauce-pipeline.adoc

```diff
@@ -0,0 +1,242 @@
+---
+layout: post
+title: "Browser-testing with Sauce OnDemand and Pipeline"
+tags:
+- pipeline
+- plugins
+- saucelabs
+- selenium
+- nightwatch
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at Cloudbees.
+
+Testing web applications across multiple browsers on different platforms can be challenging even for smaller applications.
+With Jenkins and the
+link:https://wiki.jenkins-ci.org/display/JENKINS/Sauce+OnDemand+Plugin[Sauce OnDemand Plugin],
+you can wrangle that complexity by defining your Pipeline as Code.
```

Comment:
@rtyler  - Taking your suggestion verbatim as a starting point.  

And it is "Sauce OnDemand" - that is the name of the product.